avocado/utils/cpu.py

```diff
@@ -114,3 +114,13 @@ def get_cpu_arch():
         return 'x86_64'
     else:
         return 'i386'
+
+def cpu_online_map():
+    """
+    Check out the available cpu online map
```

Comment:
@ldoktor  thanks looking into this . sure this make sense to Reports cpu as list will do .